tests/test_agent.py

```diff
@@ -104,35 +104,32 @@ def test_to_graph_should_return_orgnr_() -> None:
     assert _isomorphic
 
 
-def test_to_graph_should_return_sameas() -> None:
-    """It returns a agent graph isomorphic to spec."""
-    """It returns an sameAs graph isomorphic to spec."""
-    agent = Agent()
-    agent.identifier = "http://example.com/agents/1"
-    agent.sameas = "http://example.com/agents/2"
-
-    src = """
-    @prefix dct: <http://purl.org/dc/terms/> .
-    @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-    @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-    @prefix dcat: <http://www.w3.org/ns/dcat#> .
-    @prefix foaf:  <http://xmlns.com/foaf/0.1/> .
-    @prefix owl: <http://www.w3.org/2002/07/owl#> .
-
-    <http://example.com/agents/1> a foaf:Agent ;
-        owl:sameAs <http://example.com/agents/2> ;
-
-        .
-        """
-    g1 = Graph().parse(data=agent.to_rdf(), format="turtle")
-    g2 = Graph().parse(data=src, format="turtle")
-
-    _isomorphic = isomorphic(g1, g2)
-    if not _isomorphic:
-        _dump_diff(g1, g2)
-        pass
-    assert _isomorphic
-
+# def test_to_graph_should_return_sameas() -> None:
```

Comment:
Ikke commit utkommentert kode, fjern hvis det ikke skal med

Reply:
Den skal være med, men det trengs å implementere støtte for owl:sameAs i datacatalogtordf. :grin:

Reply:
Fjern.

Kan legge det til når du oppdaterer til en versjon av datacatalogtordf som støtter sameAs, før er det bare rot.

Reply:
Da glemmer jeg det bare. :-)

Reply:
Det er jo akkurat derfor jeg vil du skal fjerne det, utkommentert kode har en stygg tendens til å bare bli liggende...